src/App.js

```diff
@@ -1,25 +1,24 @@
-import logo from './logo.svg';
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Nav from './components/Nav';
+import Articles from './components/Articles';
+import Topics from './components/Topics';
+import SingleArticle from './components/SingleArticle';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+	return (
+		<div className='App'>
+			<Header />
+			<Nav />
+			<Routes>
+				<Route path='/' element={<Articles />} />
+				<Route path='/articles' element={<Articles />} />
+				<Route path='/articles/:article_id' element={<SingleArticle />} />
+				<Route path='/topics' element={<Topics />} />
+			</Routes>
+		</div>
+	);
 }
```

Comment:
Due to missing components above, there are some errors being thrown relating to the inability of a route to recognize a component that we would like to render at that specific endpoint.